src/services/lnd/utils-ln-service.ts

```diff
@@ -0,0 +1,35 @@
+import lnService from "ln-service"
+
+import { getLnds } from "./utils"
+
+const lndsParamsAuth = getLnds({ type: "offchain", active: true })
+
+const lnd1 = lndsParamsAuth[0].lnd
+const lnd2 = lndsParamsAuth[1].lnd
+
+interface Channel {
+  pending_payments: number[]
+}
+
+async function getPendingPaymentCountLnd1(): Promise<number> {
+  const channels = await lnService.getChannels({ lnd1 })
+  const pendingPayments = channels.filter(
+    (channel: Channel) => channel.pending_payments.length > 0,
+  )
+  return pendingPayments.length
+}
```

Comment:
this function is wrong. it just counts how many channels have active htlc. it doens't count how many pending htlcs there can be on each channel.

Reply:
addressed in https://github.com/GaloyMoney/galoy/pull/2057/commits/5559d221d8a752cdeef26ad6920f02547826352b